api/v1alpha1/condition_consts.go

```diff
@@ -57,6 +57,20 @@ const (
 	GuestCustomizationFailedReason = "GuestCustomizationFailed"
 )
 
+const (
+	// VirtualMachineToolsCondition exposes the status of VirtualMachine tools running in the guest OS, when available
+	VirtualMachineToolsCondition ConditionType = "VirtualMachineTools"
+
+	// VirtualMachineToolsNotRunningReason (Severity=Error) documents that VirtualMachine tools is not running
+	VirtualMachineToolsNotRunningReason = "VirtualMachineToolsNotRunning"
+
+	// VirtualMachineToolsRunningReason (Severity=Info) documents that VirtualMachine tools is running
+	VirtualMachineToolsRunningReason = "VirtualMachineToolsRunning"
+
+	// VirtualMachineToolsExecutingScriptsReason (Severity=Info) documents that VirtualMachine tools is starting
```

Comment:
I would leave this state out (we don't need to mirror the vmodl api, and just having running/notrunning seems cleaner and sufficient)